sormas-api/src/main/resources/strings.properties

```diff
@@ -558,12 +563,17 @@ infoUserEmail = Used to send E-Mail notifications.
-infoMergingMergeDescription = Choose this option only if you are <b>sure that the two cases are the same!</b> You should click on this button next to the case with the most recent or more complete information. Click on the IDs of the cases to open them in new tabs in order to view all their data. When the case you chose is missing information that is present in the other case, it will be updated. Also, contacts, samples, tasks and case management information will be transfered. Afterwards, the case you did not choose will be deleted.
-infoMergingPickDescription = Clicking on this button will simply delete the case that was not picked. However, no information will be copied to the case that remains active. All contacts, samples and tasks will be lost as well. Choose this option if you are sure that both cases are the same, but the other case has not been updated with any information that needs to be transfered to the one that remains active.
+infoMergingCaseMergeDescription = Choose this option only if you are <b>sure that the two cases are the same!</b> You should click on this button next to the case with the most recent or more complete information. Click on the IDs of the cases to open them in new tabs in order to view all their data. When the case you chose is missing information that is present in the other case, it will be updated. Also, contacts, samples, tasks and case management information will be transfered. Afterwards, the case you did not choose will be deleted.
+infoMergingContactMergeDescription = Choose this option only if you are <b>sure that the two contacts are the same!</b> You should click on this button next to the case with the most recent or more complete information. Click on the IDs of the cases to open them in new tabs in order to view all their data. When the case you chose is missing information that is present in the other case, it will be updated. Also, contacts, samples, tasks and case management information will be transfered. Afterwards, the case you did not choose will be deleted.
```

Comment:
Please go over this text again and make sure that it references contacts, not cases and that the content actually makes sense (specifically regarding the sentence "Also, contacts, samples, tasks and case management information will be transfered")